db/migrate/20151117165756_add_automatically_unpin_topics_to_users.rb

```diff
@@ -2,6 +2,6 @@
 
 class AddAutomaticallyUnpinTopicsToUsers < ActiveRecord::Migration[4.2]
   def change
-    add_column :users, :automatically_unpin_topics, :boolean, nullabe: false, default: true
+    add_column :users, :automatically_unpin_topics, :boolean, nullable: false, default: true
```

Comment:
https://github.com/discourse/discourse/pull/12779#discussion_r617139678:
I'm not sure what the old or new behavior are doing here. Since it's a migration, this probably indicates it failed and a new migration would be required to achieve the goal desired by the code here.

Reply:
`nullabe` was an invalid option that got ignored. Looks like we still ended up with the correct column definition. Anyway, we should change it to the following:

```suggestion
    add_column :users, :automatically_unpin_topics, :boolean, null: false, default: true
```